### DISCUSSION TITLE
Any BBC World News (BBC News) live stream?

### DISCUSSION THREAD

**Discussion by MALUrbina** [iptv-org/community#992] [Q&A]
Any one with a live stream for BBC News (BBC World News)? There is one floating here I believe but its in SD. Any HD available? Thanks

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/5165b3b06d6391e19ae8134885593845d96e3918/streams/uk_bbc.m3u#L24-L35
